[AGM-2225] Update AGVPN for iOS overview.md - Exclusions (#232)
AdguardTeam/KnowledgeBaseVPN: docs/adguard-vpn-for-ios/overview.md

## Exclusions

You can find the Exclusions tab by tapping the second icon in the bottom bar.

> By default, AdGuard VPN is active for all websites except those added to exclusions. You can also turn it on selectively — VPN will be active only for specified websites. To switch to this mode, tap *Change mode*.

Reviewer: ```suggestion
By default, AdGuard VPN is active for all websites except those added to Exclusions. You can also turn it on selectively — VPN will be active only for specified websites. To switch to this mode, tap *Change mode*.
```

> > Note that each mode has its own exclusion list.

Reviewer: замени на адмонишн

You can add domains (e.g., `google.com`) or subdomains (e.g., `news.google.com`) of websites to the lists in three ways:

- Enter them manually in the app
- [Choose from domain lists for popular services](#domain-lists-for-popular-services)

> - Add them directly from the browser by sharing the desired pages to AdGuard VPN app

Reviewer: ```suggestion
- Add them directly from the browser via the Ext Exclusions Action
```

Reviewer: а из хрома или других браузеров интересно можно?

Author: можно, ага. note: выяснили что в свежайшей 2.4 текст кнопки в шер меню - "AdGuard VPN"

![Exclusions *mobile_border](https://cdn.adtidy.org/blog/new/lpjs7iexclusions-as-of-v2-4.png)

> When you manually add a domain, all subdomains are added automatically (it looks like `*.google.com`). Want to keep only google.com or turn VPN off for a specific domain? Tap the line with the website — it will open a separate screen where you can manage subdomains. Just uncheck the boxes for those you don't need.

Reviewer: ```suggestion
When you manually add a domain, all subdomains are added automatically (it looks like `*.google.com`). Want to keep only google.com or turn VPN off for a specific domain? Tap the line with the website — this will open a separate screen where you can manage subdomains. Just uncheck the boxes for the ones you don't need.
```

![Subdomains *mobile_border](https://cdn.adtidy.org/content/release_notes/vpn/ios/v2.4/domains_en.png)

If you manually add a subdomain, the main domain will also added — but the checkbox next to it will not be checked.

### Domain lists for popular services

> Large services like Google are tricky. They usually use more than one domain — one for media files, another for the API, plus domains for different countries. For example, in the case of Google, we have 416 domains in our list — all of them need to be added to exclusions to completely turn VPN off (or on) for the service.

Reviewer: ```suggestion
Large services like Google are tricky. They usually use more than one domain — one for media files, another for the API, plus domains for different countries. For example, in the case of Google, we have 416 domains in our list — all of them need to be added to Exclusions to completely turn VPN off (or on) for the service.
```

> Here's how to add all necessary domains for a popular service to exclusions list:

Reviewer: до этого был exclusion list(s) несколько раз
можно, чтобы не путаться, писать list(s) of exclusions

Author: поправил на exclusion list
